.github/workflows/scripts/pr_labeler.py

```diff
@@ -0,0 +1,77 @@
+import os
+import sys
+from github import Github
+from github.PullRequest import PullRequest
+from github.Repository import Repository
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+def get_pr_size(pr: PullRequest) -> int:
+    excluded_files: set[str] = {".lock", ".md"}
+    additions: int = 0
+
+    for file in pr.get_files():
+        if not any(file.filename.endswith(pattern) for pattern in excluded_files):
+            additions += file.additions + file.deletions
+
+    LOGGER.info(f"PR size: {additions}")
+    return additions
+
+
+def get_size_label(size: int) -> str:
+    size_labels: dict[tuple[int, int], str] = {
+        (0, 20): "size/xs",
+        (21, 50): "size/s",
+        (51, 100): "size/m",
+        (101, 300): "size/l",
+        (301, sys.maxsize): "size/xl",
+    }
+
+    for (min_size, max_size), label in size_labels.items():
+        if min_size <= size <= max_size:
+            return label
+    return "size/xl"
```

Comment:
Do we need this fallback, since we already have `(301, sys.maxsize): "size/xl",`? If so, we can save and reuse `size/xl`

Reply:
fallback to a case where size > sys.maxsize. not sure we need it :) wdyt?

Reply:
sys.maxsize is 9223372036854775807, I hope we don't need it :laughing: